src/Controls/src/Core/Platform/Android/Extensions/FormattedStringExtensions.cs

```diff
@@ -180,42 +177,47 @@ public static void RecalculateSpanPositions(this TextView textView, Label elemen
 				var startSpan = spans[0];
 				var endSpan = spans[spans.Length - 1];
 
-				var startSpanOffset = spannableString.GetSpanStart(startSpan);
```

Comment:
There is a lot of changes with variable names that makes this a harder diff to read. Is this needed or can we revert to make it easier and better for backporting?